lib/beaker-hostgenerator/data.rb

```diff
@@ -13,7 +13,7 @@ module BeakerHostGenerator
   # `include BeakerHostGenerator::Data` and then `<function>()`.
   module Data
     module_function
-    MAIN_PE_VERSION=2021.0
+    MAIN_PE_VERSION = ENV['main_pe_version'] || '2023.0'
```

Comment:
why do you update it to 2023.0? Is that an internal beta/alpha version?

Reply:
This will be the next STS version (as of now anyway) which is what all our `main` branches are.